package.json

```diff
@@ -13,7 +13,7 @@
     "build": "rimraf dist && rollup -c rollup.config.js",
     "lint": "eslint --ext .js,.ts --ignore-path .gitignore --max-warnings 0 .",
     "lint:fix": "yarn lint --fix",
-    "test": "jest",
+    "test": "jest --coverage",
```

Comment:
Maybe we could add a separate script for running tests with coverage and call it like `test:coverage` ?